BP/common/landed_titles/BP_titles_eastern.txt

```diff
@@ -0,0 +1,51 @@
+﻿@correct_culture_primary_score = 100
+@better_than_the_alternatives_score = 50
+@always_primary_score = 1000
+
+#k_western_protectorate = {
```

Comment:
You don't need to add this one since it is from vanilla CK2.

Reply:
I don't know it might be necessary; the map extension goes further east, which seems to overlap with the lands the Northern Wei/ Norhtern Zhou/Sui/Tang occupied. So we may have to model something like that.

I'm using these maps as reference:
476: https://upload.wikimedia.org/wikipedia/commons/0/0d/East-Hem_476ad.jpg
500: https://upload.wikimedia.org/wikipedia/commons/2/2b/East-Hem_500ad.jpg
565: https://upload.wikimedia.org/wikipedia/commons/0/0f/East-Hem_565ad.jpg
600: https://upload.wikimedia.org/wikipedia/commons/a/af/East-Hem_600ad.jpg
700: https://upload.wikimedia.org/wikipedia/commons/2/21/East-Hem_700ad.jpg

Reply:
Yes, but if those realms are on the map I don't see where the Western Protectorate would fit in.

Reply:
Well, the eastern edge of the map lies within various Chinese dynasties. The CK2 map ended roughly where k_tuyuhun/d_qamdo holders are in this branch. So we need something to model the Chinese presence on this map. In CK3 vanilla, they do that with k_guiyi. So maybe some set of titles like k_western_northern_wei/k_western_northern_zhou/&c. for the history?

Reply:
Yes, or if there are sub-vassals that would be appropriate. But I don't think it would be the Western Protectorate, although it is not a major issue to include it for now

Reply:
Then I'll refactor to be k_western_<dynasty_name> when I fill in that portion of the history.

Reply:
Ok, fair enough

Reply:
Replaced k_western_protectorate with k_northern_wei_western_regions for the 476 start. We can add additional Chinese titles when more bookmarks are added I think.